src/Components/API/apiCalls.ts

```diff
@@ -0,0 +1,12 @@
+const fetchRandom = (): Promise<any> => {
```

Comment:
I'm not sure why entirely but I had read that we should try to avoid the any type.